website/content/docs/connect/cluster-peering/usage/k8s.mdx

```diff
@@ -0,0 +1,231 @@
+  $ export HELM_RELEASE_NAME=cluster-02
+  $ helm install ${HELM_RELEASE_NAME} hashicorp/consul --create-namespace --namespace consul --version "1.0.1" --values values.yaml --set global.datacenter=dc2 --kube-context $CLUSTER2_CONTEXT
+  ```
+
+## Mesh gateway configuration for Kubernetes
+
+Mesh gateways are required for cluster peering connections. To add mesh gateways to your deployment in order to use cluster peering complete the following steps:
+
+1. In `cluster-01` create the `Mesh` custom resource with `peeringThroughMeshGateways` set to `true`.
+
+    <CodeBlockConfig filename="mesh.yaml">
+
+    ```yaml
+    apiVersion: consul.hashicorp.com/v1alpha1
+    kind: Mesh
+    metadata:
+      name: mesh
+    spec:
+      peering:
+        peerThroughMeshGateways: true
+    ```
+
+    </CodeBlockConfig>
+
+1. Apply the mesh gateway to `cluster-01`.
+
+    ```shell-session
+    $  kubectl --context $CLUSTER1_CONTEXT apply -f mesh.yaml 
+    ```
+
+1. Repeat the process to create and apply a mesh gateway with cluster peering enabled to `cluster-02`:
+
+    <CodeBlockConfig filename="mesh.yaml">
+
+    ```yaml
+    apiVersion: consul.hashicorp.com/v1alpha1
+    kind: Mesh
+    metadata:
+      name: mesh
+    spec:
+      peering:
+        peerThroughMeshGateways: true
+    ```
+
+    </CodeBlockConfig>
+
+    ```shell-session
+    $  kubectl --context $CLUSTER2_CONTEXT apply -f mesh.yaml 
+    ```
+
+## Recreate or reset a peering connection
+
+To recreate or reset the peering connection, you need to generate a new peering token from the cluster where you created the `PeeringAcceptor`.
+
+1. In the `PeeringAcceptor` CRD, add the annotation `consul.hashicorp.com/peering-version`. If the annotation already exists, update its value to a higher version.
+
+    <CodeBlockConfig filename="acceptor.yml" highlight="6" hideClipboard>
+
+    ```yaml
+    apiVersion: consul.hashicorp.com/v1alpha1
+    kind: PeeringAcceptor
+    metadata:
+      name: cluster-02
+      annotations:
+        consul.hashicorp.com/peering-version: "1" ## The peering version you want to set, must be in quotes
+    spec:
+      peer:
+        secret:
+          name: "peering-token"
+          key: "data"
+          backend: "kubernetes"
+    ```
+
+    </CodeBlockConfig>
+
+1. After updating `PeeringAcceptor`, repeat the following steps to create a peering connection:
+      1. [Create a peering token](#create-a-peering-token)
+      1. [Establish a peering connection between clusters](#establish-a-peering-connection-between-clusters)
+      1. [Export services between clusters](#export-services-between-clusters)
+      1. [Authorize services for peers](#authorize-services-for-peers)
+
+      Your peering connection is re-established with the updated token.
+
+~> **Note:** The only way to create or set a new peering token is to manually adjust the value of the annotation `consul.hashicorp.com/peering-version`. Creating a new token causes the previous token to expire.
```

Comment:
```suggestion
~> **Compatibility warning:** The only way to create or set a new peering token is to manually adjust the value of the annotation `consul.hashicorp.com/peering-version`. Creating a new token causes the previous token to expire.
```
I'm not sure that this elevates to the level of warning or if this is right category. I would just take it out of the callout.